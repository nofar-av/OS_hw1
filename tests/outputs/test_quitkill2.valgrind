==28250== Memcheck, a memory error detector
==28250== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28250== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28250== Command: /home/omer/OS_wet1/smash
==28250== Parent PID: 28245
==28250== 
==28250== 
==28250== FILE DESCRIPTORS: 4 open at exit.
==28250== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_quitkill2.valgrind
==28250==    <inherited from parent>
==28250== 
==28250== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_quitkill2.err
==28250==    <inherited from parent>
==28250== 
==28250== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_quitkill2.out
==28250==    <inherited from parent>
==28250== 
==28250== Open file descriptor 0:
==28250==    <inherited from parent>
==28250== 
==28250== 
==28250== HEAP SUMMARY:
==28250==     in use at exit: 0 bytes in 0 blocks
==28250==   total heap usage: 25 allocs, 25 frees, 82,072 bytes allocated
==28250== 
==28250== All heap blocks were freed -- no leaks are possible
==28250== 
==28250== For counts of detected and suppressed errors, rerun with: -v
==28250== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
