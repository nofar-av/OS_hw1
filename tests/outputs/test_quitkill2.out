smash> smash> smash> smash: sending SIGKILL signal to 2 jobs:
5893: sleep 10000&
5894: sleep 10000&
